napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -295,11 +295,30 @@ def _set_table(self, layer_str: str = ''):
     def _get_layer_actions(self):
```

Comment:
Maybe we could update the name of this function and add a docstring to aid readability? More accurately it gives you all actions you want to avoid conflict with. I cannot think of a good short name though!
Maybe something like `_get_conflict_avoid_actions' ( :sweat_smile: this seems too long)?

If we can't think of a good name, lets just at least add a docstring.

Reply:
Yep, I was also thinking that the method needs a better name but I was as well unable to come with something that worked better 😅 Will add a doctring :+1:

Reply:
I would say `get_conflicting_actions` or `get_potential_conflicting_actions`. I don't mind if it's long, personally. Definitely we shouldn't leave it as is though haha